Libraries/LibWeb/CSS/Parser/Parser.h

```diff
@@ -58,6 +58,14 @@ struct NegateNode {
 };
 }
 
+// The very large CSS Parser implementation code is broken up among several .cpp files:
```

Comment:
This comment seems extremely likely to get out of date, and unlikely to ever by updated past today.

Perhaps it's best to move the architecture split description to a markdown file in Documentation?

Reply:
Ehh, I get your point, but I feel like putting things in Documentation is worse for out-of-dateness, just because it's further from the code itself.

Maybe I don't need this comment at all. I don't think the previously split-out files have confused anyone?

Reply:
I trimmed it down. Should be enough without danger of it getting outdated.